sdccc/src/main/java/com/draeger/medical/sdccc/manipulation/GRpcManipulations.java

```diff
@@ -206,16 +206,16 @@ public Optional<String> createContextStateWithAssociation(
                 response -> response.getStatus().getResult(),
                 msg -> {
                     if (msg.getContextStateHandle().isBlank()) {
-                        return Optional.empty();
+                        return ManipulationResponse.from(msg.getStatus(), Optional.empty());
```

Comment:
I would expect: 
return ManipulationResponse.from(msg.getStatus().**getResult**(), Optional.empty());

Reply:
It's the same thing, `.from()` can be used for `BasicResponses.BasicResponse` and for `ResponseTypes.Result`